arch/s390x-options.mk

```diff
@@ -8,5 +8,6 @@
 MACHINETYPE := s390-ccw-virtio
 KERNELPARAMS :=
 MACHINEACCELERATORS :=
+CPUFEATURES := pmu=off
```

Comment:
This is a feature change for os390 so we'd better get input from @alicefr.

Reply:
`pmu` is not implemented and you can an error if you try to use this option. For example:
```sh
qemu-system-s390x -cpu host,pmu=off
qemu-system-s390x: can't apply global host-s390x-cpu.pmu=off: Property '.pmu' not found
```

Reply:
Thanks @alicefr! Please can yuo remove this part of the change @justin-he?

Reply:
ok, thanks for the confirming, I will remove it for s390